trashtalk/__init__.py

```diff
@@ -15,11 +15,9 @@
 from trashtalk.factories import app_factory
 
 
-app = Flask(__name__)
-
 # Default configuration is development!
 # Override with config.from_envvar('APP_ENV_VAR')
 # http://flask.pocoo.org/docs/0.12/config/#configuring-from-files
 # app.config.from_object('trashtalk.settings.Development')
-app = app_factory('trashtalk.settings.Development')
-app.logger.info('Welcome to the Development server')
+# app = app_factory('trashtalk.settings.Development')
```

Comment:
Old stuff that should have been deleted in a previous update! These lines of code haven't been in use for a while now.